test/Libraries/DynamoPythonTests/PythonEvalTests.cs

```diff
@@ -15,7 +16,7 @@ public class PythonEvalTests
             DSCPython.CPythonEvaluator.EvaluatePythonScript,
             DSIronPython.IronPythonEvaluator.EvaluateIronPythonScript
         };
-
+        static int count = 0;
```

Comment:
Could you instead define the handler as a delegate/lambda internal to the test so there is no need to define a static field that is only to be used by that test?

Reply:
I don't want to use a lambda because then I can't unsubscribe, but I will try a delegate.

Reply:
Well, a lambda is a `Func` which pretty much works in the same way as a delegate. As long as you have a variable assignment to it you can unsubscribe it.